virtwho/manager/satellite/satellite.py

```diff
@@ -91,7 +91,17 @@ def _register_system(self, hypervisor_uuid, hypervisor_type, systemid_filename):
             raise SatelliteError("Unable to login to satellite5 server: %s" % str(e))
 
         try:
-            hypervisor_base_channel = self.server_rpcapi.channel.software.getDetails(session, 'hypervisor-base')
+            userdetail = self.server_rpcapi.user.getDetails(session, self.username)
+            org_id = userdetail["org_id"]
+        except Exception as e:
+            self.logger.exception("Unable to get user details")
+            raise SatelliteError("Unable to get user details %s" % str(e))
```

Comment:
Suggestion: Perhaps add a colon between "Unable to get user details" and the inserted string.